ArcDPS Boon Table/main.cpp

```diff
@@ -187,11 +213,30 @@ uintptr_t mod_wnd(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
 	return uMsg;
 }
 
+constexpr auto num_of_npcs = 3;
+
+
+//TODO: Figure something better out, maybe some id, like for bosses, is shown somewhere?
+// this array only needs to contain the beginning of the name
+const std::string npc_names[num_of_npcs] = {
+	"Priory Scholar", //Glenna (Wing 3, 1st encounter: Escort)
+	"Saul", //Saul D'Alessio (Wing 4, 4th encounter: Deimos)
+	"Desmina" //Desmina?, River of Souls
+};
+bool npc_registered[num_of_npcs] = { false };
+
+uintptr_t npc_ids[num_of_npcs];
+
+
+
+
 /* combat callback -- may be called asynchronously. return ignored */
 /* one participant will be party/squad, or minion of. no spawn statechange events. despawn statechange only on marked boss npcs */
 uintptr_t mod_combat(cbtevent* ev, ag* src, ag* dst, char* skillname, uint64_t id, uint64_t revision)
 {
-	Player* current_player = nullptr;
+	Entity* current_entity = nullptr;
+	char buff[4096];
+	char* p = &buff[0];
```

Comment:
Remove all the code, used to output the events. We don't need the example plugin in here, it will just make everything a little bit slower.

Reply:
Removed all the logs, forgot they were still in there